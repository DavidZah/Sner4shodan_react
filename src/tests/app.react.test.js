import React from 'react';  
import  AppRouter from '../app';
import renderer from 'react-test-renderer';  

it('renders correctly', () => {  
const tree = renderer.create(<AppRouter />).toJSON();  
  expect(tree).toMatchSnapshot();  
});

