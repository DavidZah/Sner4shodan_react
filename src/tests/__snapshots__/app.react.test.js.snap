// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="navbar navbar-inverse navbar-static-top"
  >
    <div
      className="container-fluid"
    >
      <div
        className="navbar-header"
      >
        <p
          className="navbar-brand"
        >
          Sner 4 Shodan
        </p>
      </div>
      <div
        className="collapse navbar-collapse"
        id="navbar"
      >
        <ul
          className="nav navbar-nav"
        >
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
               Home 
            </a>
          </li>
          <li>
            <a
              href="/hosts/"
              onClick={[Function]}
            >
               Hosts 
            </a>
          </li>
          <li>
            <a
              href="/services/"
              onClick={[Function]}
            >
               Services 
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>,
  <div>
    <div
      className="card-body"
    >
      <h2>
        Microsoft
      </h2>
      <div>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              0
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              0
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            samba
          </div>
          <div
            className="proto"
          >
            Proto:
            udp
          </div>
          <div
            className="port"
          >
            Port:
            666
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            samba old ver
             
          </div>
          <div
            className="type"
          >
            Commnet:
            need update
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              1
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              0
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            ssh
          </div>
          <div
            className="proto"
          >
            Proto:
            tcp
          </div>
          <div
            className="port"
          >
            Port:
            23
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            active ssh tunel
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try admin admin
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              2
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              0
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            http
          </div>
          <div
            className="proto"
          >
            Proto:
            tcp
          </div>
          <div
            className="port"
          >
            Port:
            80
             
          </div>
          <div
            className="state"
          >
            State:
            deactivate
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            not secure internet connection
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try https
             
          </div>
        </pre>
      </div>
    </div>
    <div
      className="card-body"
    >
      <h2>
        Sony
      </h2>
      <div>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              0
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              1
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            https
          </div>
          <div
            className="proto"
          >
            Proto:
            udp
          </div>
          <div
            className="port"
          >
            Port:
            443
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            secure
             
          </div>
          <div
            className="type"
          >
            Commnet:
            i like it
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              1
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              1
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            ftp
          </div>
          <div
            className="proto"
          >
            Proto:
            tcp
          </div>
          <div
            className="port"
          >
            Port:
            20
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            file transfare protocol
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try paswd 123456789
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              2
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              1
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            dns
          </div>
          <div
            className="proto"
          >
            Proto:
            tcp
          </div>
          <div
            className="port"
          >
            Port:
            53
             
          </div>
          <div
            className="state"
          >
            State:
            deactivate
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            domain name sever
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try hack it
             
          </div>
        </pre>
      </div>
    </div>
    <div
      className="card-body"
    >
      <h2>
        NASA
      </h2>
      <div>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              0
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              2
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            https
          </div>
          <div
            className="proto"
          >
            Proto:
            tcp
          </div>
          <div
            className="port"
          >
            Port:
            443
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            secure
             
          </div>
          <div
            className="type"
          >
            Commnet:
            i like it
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              1
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              2
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            ftp
          </div>
          <div
            className="proto"
          >
            Proto:
            what that mean
          </div>
          <div
            className="port"
          >
            Port:
            20
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            file transfare protocol
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try paswd 123456789
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              2
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              2
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            dns
          </div>
          <div
            className="proto"
          >
            Proto:
            i dont know
          </div>
          <div
            className="port"
          >
            Port:
            53
             
          </div>
          <div
            className="state"
          >
            State:
            deactivate
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            domain name sever
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try hack it
             
          </div>
        </pre>
        <pre>
          <h3>
            <span
              className="id"
            >
              Id:
              3
               
            </span>
            <span
              className="host_id"
            >
              Host id:
              2
            </span>
          </h3>
          <div
            className="service"
          >
            Name:
            dns
          </div>
          <div
            className="proto"
          >
            Proto:
            tcp
          </div>
          <div
            className="port"
          >
            Port:
            40
             
          </div>
          <div
            className="state"
          >
            State:
            active
             
          </div>
          <div
            className="type"
          >
            Type:
             
          </div>
          <div
            className="type"
          >
            Info:
            domain name sever
             
          </div>
          <div
            className="type"
          >
            Commnet:
            try hack it
             
          </div>
        </pre>
      </div>
    </div>
  </div>,
]
`;
